packages/shared/src/components/Logo.tsx

```diff
@@ -72,19 +75,20 @@ export default function Logo({
 }: LogoProps): ReactElement {
   return (
     <LinkWithTooltip
-      href={process.env.NEXT_PUBLIC_WEBAPP_URL}
+      href={homeHref}
       passHref
       prefetch={false}
       tooltip={{ placement: 'right', content: 'Home' }}
     >
-      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
       <a
+        aria-disabled={linkDisabled}
```

Comment:
Should we also add `disabled={linkDisabled}`? Just so the behavior would work as how it is explained by the aria.

Reply:
I've tried but linter suggested to remove because we should not use it in links